HW_4/HW_4.py

```diff
@@ -0,0 +1,69 @@
+import requests
+import time
+import os
+import threading
+from multiprocessing import Process
+import asyncio
+import aiohttp
+
+urls = ['https://cdn.pixabay.com/photo/2024/01/07/14/12/man-8493244_1280.jpg',
+        'https://cdn.pixabay.com/photo/2023/09/13/13/48/cactus-8250996_1280.jpg',
+        'https://cdn.pixabay.com/photo/2023/10/17/03/23/child-8320341_1280.png'
+        ]
+start_time = time.time()
+folder = 'data_task'
+if not os.path.exists(folder):
+    os.mkdir(folder)
+
+
+def download(url):
+    response = requests.get(url)
+    file = os.path.join(folder, url.split('/')[-1])
+    with open(file, "w", encoding='utf-8') as f:
+        f.write(response.text)
```

Comment:
А вы не заметили, что скачиваемые вами картинки, не открываются?
```suggestion
    with open(file, "wb") as f:
        f.write(response.content)
```